src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/colors.fluent.scss

```diff
@@ -38,3 +39,5 @@ $red180: #a80000;
 
 $yellow20: #fff4ce;
 
+$white: #ffffff;
```

Comment:
The variable $white is defined before in line 7

Reply:
Removed duplicate variable